sdk/synapse/azure-synapse/azure/synapse/_configuration.py

```diff
@@ -0,0 +1,68 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import Any
+
+from azure.core.configuration import Configuration
+from azure.core.pipeline import policies
+
+from ._version import VERSION
+
+
+class SynapseClientConfiguration(Configuration):
+    """Configuration for SynapseClient
+    Note that all parameters used to create this instance are saved as instance
+    attributes.
+
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: azure.core.credentials.TokenCredential
+    :param synapse_dns_suffix: Gets the DNS suffix used as the base for all Synapse service requests.
+    :type synapse_dns_suffix: str
+    :param livy_api_version: Valid api-version for the request.
+    :type livy_api_version: str
+    """
+
+    def __init__(
+        self,
+        credential,  # type: "TokenCredential"
+        synapse_dns_suffix,  # type: str
```

Comment:
Can we set default value "dev.azuresynapse.net" for synapse_dns_suffix? Just like livy_api_version

Reply:
fixed